doc/jsk_pcl_ros/nodes/euclidean_clustering.md

```diff
@@ -12,7 +12,12 @@ tries to track the segment.
 ## Subscribing Topics
 * `~input` (`sensor_msgs/PointCloud2`):
 
-   input pointcloud.
+   input pointcloud. If `multi` is `talse`, this input is only enough.
+
+
+* `~input/indices` (`jsk_recognition_msgs/ClusterPointIndices`):
```

Comment:
I think it is better to use `~input/cluster_indices` because `mask_rcnn` node's output is `~output/cluster_indices`.

Reply:
OK. I modified it.